.github/workflows/release.yml

```diff
@@ -2,8 +2,14 @@ name: Release
 
 on: workflow_dispatch
 
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+
 jobs:
   release:
+    defaults:
+      run:
+        working-directory: frontend
```

Comment:
Does the `working-directory` not need to be set for the `release-container-image` job?

Reply:
No since the release workflow needs the scope of the package.json this is required. If our package.json and code were at top level. We wouldn't have to set work dir at all.